Zion's End/src/Tower.cpp

```diff
@@ -7,6 +7,7 @@ constexpr uint16 canonTextureWidth = 95;
 constexpr uint16 canonTextureHeight = 160;
 constexpr uint16 towerTextureSize = 256;
 constexpr float DEFAULT_COOLDOWN = 2.f;
+constexpr float DEFAULT_RANGE = 250.f;
```

Comment:
I thought about a possibility of having a tower with smaller sight radius than the range of its projectile, but maybe that doesn't make sense.